capstone/labs/templates/labs-about.md

```diff
@@ -8,8 +8,12 @@ row_style: bg-white
 
 ###[Chronolawgic]({% url "labs:chronolawgic" %})
 
-A legal timeline tool, allowing CAP users to plot caselaw alongside other
-events chrono(law)gically. Plot caselaw alongside other events to understand the progression of U.S. caselaw.
-Must have an account in order to set up a timeline (create an account [here]({% url 'register' %})).
+A legal timeline tool that allows CAP users to plot caselaw alongside other
+events chrono(law)gically. 
```

Comment:
I like the expression "chrono(law)gically" except it doesn't make sense without the parenthetic infix. I think I'd just use "chronologically" here. Please forgive me if that's too uptight. ;)